community/kernel/src/main/java/org/neo4j/kernel/impl/logging/SimpleLogService.java

```diff
@@ -27,6 +27,12 @@
     private final LogProvider userLogProvider;
     private final LogProvider internalLogProvider;
 
+    public SimpleLogService( LogProvider commonLogProvider )
+    {
+        this.userLogProvider = commonLogProvider;
```

Comment:
So this constructor should work differently than the one below? The one below wraps in `DuplicatingLogProvider`. If they should differ then please document and explain exactly why and which you should use in different scenarios.